monkey/agent_plugins/exploiters/snmp/src/snmp_command_builder.py

```diff
@@ -29,11 +29,12 @@ def build_snmp_command(
 
     monkey_cmd = build_monkey_commandline(agent_id, servers, current_depth + 1)
 
-    return SNMP_LINUX_COMMAND_TEMPLATE % {
+    command = SNMP_LINUX_COMMAND_TEMPLATE % {
         "monkey_path": get_agent_dst_path(target_host),
         "http_path": agent_download_url,
         "monkey_type": MONKEY_ARG,
         "parameters": monkey_cmd,
         "agent_otp_environment_variable": AGENT_OTP_ENVIRONMENT_VARIABLE,
         "agent_otp": otp,
     }
+    return f'-c "{command}"'
```

Comment:
Should the "-c" just be added to `SNMP_LINUX_COMMAND_TEMPLATE`?

Reply:
My reasoning was that it was a nice clear way to show that the command needed to be quoted.

This is going to be reworked as we work to shrink the commands to fit the 255 character limit imposed by the MIB

Reply:
Note: We also need to be sure to run the wget command in silent mode. We will probably have to silence the agent's output as well